types/2020-03-02/Issuing/Authorizations.d.ts

```diff
@@ -103,17 +103,17 @@ declare module 'stripe' {
         metadata: Metadata;
 
         /**
-         * The amount the user is requesting to be authorized. This field will only be non-zero during an `issuing.authorization.request` webhook.
+         * [DEPRECATED] The amount the user is requesting to be authorized. This field will only be non-zero during an `issuing_authorization.request` webhook.
          */
         pending_authorized_amount?: number;
 
         /**
-         * The additional amount Stripe will hold if the authorization is approved. This field will only be non-zero during an `issuing.authorization.request` webhook.
+         * [DEPRECATED] The additional amount Stripe will hold if the authorization is approved. This field will only be non-zero during an `issuing_authorization.request` webhook.
```

Comment:
Ideally we'd say what to do instead of the deprecated thing, but not a part of this PR.